src/KlarnaOrderManagement.php

```diff
@@ -111,6 +111,7 @@ function () {
 		$this->settings = new Settings();
 		$this->metabox  = new MetaBox( $this );
 		$this->ajax     = new Ajax();
+		new ReturnFee();
```

Comment:
Suggestion: This should be set to a class property instead of just creating an instance. If we ever need to unhook actions attached to the instance for some reason then this will be very hard to do with how this is now.